packages/amplify-cli/src/commands/plugin/configure.ts

```diff
@@ -93,7 +93,7 @@ Only explicitly added plugins are active.');
 function displayPluginDirectoriesLearnMore(context: Context) {
   context.print.info('');
   context.print.green('The directories contained this list are searched for \
-plugins in a plugin scan.');
+plugins during a plugin scan.');
```

Comment:
I would change this whole sentence to 'A plugin scan searches this directory list for plugins'